app/controllers/projects_controller.rb

```diff
@@ -337,6 +337,8 @@ def default_context_filter
     if default_context_name.present?
       default_context = current_user.contexts.where(:name => default_context_name).first_or_create
       p['default_context_id'] = default_context.id
+    else
+      p['default_context_id'] = NIL
```

Comment:
using `NIL` here works, but I'd rather this would be lowercase `nil` so that people don't go looking for a constant named `NIL` in the code base.